faucetagent.py

```diff
@@ -332,7 +335,10 @@ def parse():
     arg('--key', required=True, help='private key file')
     arg('--gnmiport', type=int, default=10161, help='gNMI port (10161)')
     arg('--configfile', required=True, help='FAUCET config file')
-    arg('--promport',
+    arg('--prom_addr',
```

Comment:
I think we should be consistent with underscores in command line options; I find them inconvenient to type so my preference is to not use them in command line options. PEP8 (annoyingly) recommends their use in Python code and that also seems to be the FAUCET style, so that's why the internal variable names are different. We could change them internally for consistency (though we might potentially have to silence some pylint complaints.)